tests/storage-disks-vm

```diff
@@ -109,6 +109,18 @@ waitInstanceReady v1
 lxc exec v1 -- ls -l /mnt/foo1
 lxc stop -f v1
 
+# Check adding a disk to a vm with a long name to test possible problems with long socket paths or long qemu device tags
+LONG_DEVICE_NAME="device-with-very-long-name-for-long-qemu-property-handling-tests"
+
+lxc init "${IMAGE}" v2 --vm
+lxc config device add v2 "${LONG_DEVICE_NAME}" disk source="${testRoot}/allowed1" path="/mnt"
+lxc start v2
+waitInstanceReady v2
+lxc config device remove v2 "${LONG_DEVICE_NAME}"
+lxc config device add v2 "${LONG_DEVICE_NAME}" disk source="${testRoot}/allowed1" path="/mnt" # Test hotplugging as well
+lxc exec v2 -- ls -l /mnt
```

Comment:
I am wondering if this test is sufficient as `/mnt` is one of the directories that is always present. So even if the hotplugging fails internally the directory would still be there and the command succeeds. 

What do you think about picking something like `path=/mnt/foo` like in the test above?